algorithms/kernel/svm/svm_predict_impl.i

```diff
@@ -194,23 +194,19 @@ struct SVMPredictImpl<defaultDense, algorithmFPType, cpu> : public Kernel
             service_memset<algorithmFPType, cpu>(distance, zero, nVectors);
             return Status();
         }
-        else
-        {
-            service_memset<algorithmFPType, cpu>(distance, bias, nVectors);
-        }
         const NumericTablePtr svTable = model->getSupportVectors();
 
         ReadColumns<algorithmFPType, cpu> mtSVCoeff(*svCoeffTable, 0, 0, nSV);
         DAAL_CHECK_BLOCK_STATUS(mtSVCoeff);
-        const algorithmFPType * svCoeff = mtSVCoeff.get();
+        const algorithmFPType * const svCoeff = mtSVCoeff.get();
 
         size_t nRowsPerBlock = 1;
-        DAAL_SAFE_CPU_CALL((nRowsPerBlock = 256), (nRowsPerBlock = nVectors));
-
+        DAAL_SAFE_CPU_CALL((nRowsPerBlock = 2048), (nRowsPerBlock = nVectors));
```

Comment:
Does changing ```nRowsPerBlock``` affect performance? Do you have any measurements?

Reply:
@owerbat 
Yes. The increase in the block is associated with a change in the TLS structure (TLS -> LS). To ensure stability. But at the same time, an overhead per block has grown. To compensate for it, I had to reduce the number of blocks.

Reply:
seems it would be better to have member field of method which returns optimal block size depending on cache sizes.
 let it be for ex. :
```
size_t getBlockSize() {
  return 2048;
}
```
and when we will return to optimization we could change just this functionality.
please create task or issue for experimenting with block size for future optimizations!